html2text/__init__.py

```diff
@@ -181,11 +184,7 @@ def handle_charref(self, c):
         self.handle_data(charref, True)
 
     def handle_entityref(self, c):
```

Comment:
In the commit message you wrote: `I don't see the point of keeping HTML entities as-is during the conversion`. My question: Is it possible to pass some HTML in the job description that would break the website? e.g. css matches some tags inside the description which results in a broken layout; someone passes malicious javascript `<img src="http://url.to.file.which/not.exist" onerror=alert(document.cookie);>`